docs/fundamentals/executor/executor-reload.md

```diff
@@ -0,0 +1,94 @@
+(reload-executor)=
```

Comment:
I don't think we should add a page for this feature

Reply:
Then tell me where this should be added